peps/pep-0772.rst

```diff
@@ -0,0 +1,463 @@
+
+-----------------------
+Python Steering Council
+-----------------------
+
+While there is overlap between the PyPA, Packaging-WG as well as the Python
+core team, the Steering Council is not well-placed to make decisions over
+Python packaging matters directly. Packaging only tangentially intersects with
+Python language evolution, packaging requires specialised expertise and deep
+domain knowledge (much like typing or the C API), and there is a different
+cohort of potential electors for a Packaging Council than the Steering Council.
+
+------------
+Expectations
+------------
+
+An elected Packaging Council that has authority over the interoperability
+standards and Python packaging tooling would have the mandate to coordinate
+efforts around Python packaging. This would provide a better, more consistent
+experience to packagers, package consumers, tool developers, PyPI, and other
+index maintainers. With greater transparency and clearer goals via an elected
+Packaging Council, the PSF will also be able to offer more tactical work,
+longer term strategic vision, and fundraising support to all parts of this
+ecosystem.
+
+Some expectations around the formation of a Packaging Council are that they
+will facilitate:
+
+* Work with PSF staff and the newly formed `User Success Workgroup`_ toward
+  improving the user experience around packaging.
+* A more stable packaging ecosystem that is more responsive to community input.
+* Increased transparency and clearer shared goals for the packaging ecosystem.
+* Increased tactical and fundraising support from the PSF, to increase capacity
+  and funding available to packaging tools.
+
+=============
+Specification
+=============
+
+-----------------
+Packaging Council
+-----------------
+
+The Packaging Council will be composed of 5 individuals.
+
+Mandate
+=======
+
+The council shall work to:
+
+* Improve Python packaging's user experience.
+* Maintain the quality and stability of the Python packaging standards.
+* Formalise and maintain the relationship with the core team as well as the
+  PSF.
+* Establish appropriate decision-making processes.
+* Seek consensus among contributors before acting in a formal capacity.
+* Make contributing as accessible, inclusive, and sustainable as possible.
+
+Responsibilities
+================
+
+The council shall:
+
+* have broad authority over the Python packaging standards and Python Packaging
+  User Guide, that are maintained on packaging.python.org.
+* establish processes for making binding decisions regarding packaging
+  standards, tools and implementations as well as for considering
+  ecosystem-wide changes.
+* look for ways to use their direct authority as little as possible – seeking
+  consensus/consent over voting.
+
+To use its powers, the council votes. Every council member must either vote or
+explicitly abstain. Members with conflicts of interest on a particular vote
+must abstain. Passing requires support from a majority of non-abstaining
```

Comment:
I would recommend that a minimum quorum of 3 voting members should be used when there are multiple abstentions. If more than 2 members abstain, that begs the question is the item up for vote well scoped for decision making.

Reply:
Good question @warsaw. In general, it shouldn't be a problem. My concern is that 4 people abstain and the decision falls on 1 person's shoulders. I'm fine with mirroring PEP 13 unless necessary to diverge.